ios/Client/EXRootViewController.m

```diff
@@ -132,10 +132,21 @@ - (void)moveHomeToVisible
   }];
 }
 
+- (BOOL)_isHomeVisible {
+  return [EXKernel sharedInstance].appRegistry.homeAppRecord.viewController == [self contentViewController];
+}
+
 // this is different from Util.reload()
 // because it can work even on an errored app record (e.g. with no manifest, or with no running bridge).
 - (void)reloadVisibleApp
 {
+  // treat home differently from other apps - just reload the bridge
+  if ([self _isHomeVisible]) {
```

Comment:
Wouldn't we want to also scope this behavior to Home development? (If a user runs their simulator and uses cmd+r I don't think his Home should reload… or should it?)

Reply:
`reloadBridge` takes care of this for us: https://github.com/expo/expo/blob/714a594adb16175b6490ad488f3a97c7fbf922b1/ios/Exponent/Kernel/ReactAppManager/EXReactAppManager.m#L577-L582

i added a comment to make that clear